NEW : Duplicate ref_client from order origin to intervention.

dolibarr95 opened the pull request:
# NEW[Duplicate ref_client order origin to intervention]
When creating an intervention from an order, duplicate the ref_client (as is already done for public and private notes).

dolibarr95 committed:
Duplicate ref_client from order origin to intervention.

When creating an intervention from an order, duplicate the ref_client (as is already done for public and private notes).

eldy committed:
Merge branch 'develop' into patch-66

dolibarr95 commented:
🥳 https://www.dolibarr.fr/forum/t/et-le-prix-de-la-30000eme-pr-revient-a/46690

eldy commented on htdocs/fichinter/card.php:
```diff
@@ -882,6 +882,7 @@
 
 			$soc = $objectsrc->thirdparty;
 
+			$ref_client = (!empty($objectsrc->ref_client) ? $objectsrc->ref_client : GETPOST('ref_client', 'alpha'));
```
Imagine you crane an intervention from a proposal.
The proposal as the ref PRABC for you and the proposal as the ref PR456 on customer side (so ref_client is PR456.

If you copy the ref of proposal into the intervention, you will got the ref of intervention that is
IN123 and the reference of this intervention into the customer system of the client will ba saved as PR456.
But this is a reference of a proposal not of an intervention. You are mixing apple and pear here. So i suggest you introduce a hidden option
INTERVENTION_SET_CUSTOMER_PROPOSAL_REF_WITH_CUSTOMER_INTERVENTION_REF

dolibarr95 committed:
add hidden option

Add the INTERVENTION_SET_CUSTOMER_PROPOSAL_REF_WITH_CUSTOMER_INTERVENTION_REF hidden option under @eldy's advice.

dolibarr95 commented:
Hello
@eldy tks for reading and feedback.

dolibarr95 committed:
Merge branch 'develop' into patch-66

dolibarr95 committed:
Merge branch 'develop' into patch-66

eldy committed:
Merge branch 'develop' into patch-66

eldy committed:
Merge branch 'develop' into patch-66
